menu/views.py

```diff
@@ -11,33 +11,34 @@
 from menu.filters import CustomSearchFilter, CustomRangeFilter, perform_product_filtering
 from menu.serializers import LocationMenuSerializer, ProductMenuSerializer
 
-
 TIMEOUT = CACHE_TIMEOUT.get('LocationMenuView')
 
+
+@method_decorator(cache_page(TIMEOUT['decorator']), name='retrieve')
```

Comment:
# it's able to use decorator for class with method name as parameter

Reply:
done